CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Changelog
 
+## Unreleased
+
+### Bug fixes
+
+* Rename remaining NEWS to CHANGELOG. ([#1308])
+* Fix links in CHANGELOG. ([#1339])
+* Replace all links with http to https. ([#1340])
+* Fix have_db_column.with_options misspelled options. ([#1358])
+* Improve error message for assossiation_matcher.with_foreign_key option. ([#1376])
```

Comment:
In general it might be best to put updates to error messages — or what I would consider to be changes to "user experience" — in Improvements. You're not really fixing broken functionality, just making the software easier to use in some way.